selfdrive/car/hyundai/values.py

```diff
@@ -1375,6 +1377,23 @@ class Buttons:
       b'\xf1\x8795441-3D220\x00\xf1\x81G19_Rev\x00\x00\x00\xf1\x00PSBG2441  G19_Rev\x00\x00\x00SNX4T16XXHS01NS2lS\xdfa',
     ],
   },
+  CAR.KIA_SPORTAGE_HYBRID_5TH_GEN: {
+    (Ecu.fwdCamera, 0x7c4, None): [
+      b'\xf1\x00NQ5 FR_CMR AT USA LHD 1.00 1.00 99211-P1060 665',
+    ],
+    (Ecu.eps, 0x7d4, None): [
+      b'\xf1\x00NQ5 MDPS C 1.00 1.04 56300-CH000 2322',
+    ],
+    (Ecu.engine, 0x7e0, None): [
+      b'\xf1\x87391312MAD0',
+    ],
+    (Ecu.transmission, 0x7e1, None): [
+      b'\xf1\x00PSBG2441  G19\x00\x00\x00\x00\x00\x00\x00SNQ5T16XXHS01NS04\xdc_\xfb',
+    ],
+    (Ecu.fwdRadar, 0x7d0, None): [
+      b'\xf1\x00NQ5__               1.01 1.03 99110-CH000         ',
+    ],
+  },
```

Comment:
Just a heads up, all CAN-FD cars now only fingerprint with fwdCamera and fwdRadar

Reply:
@sshane can you add a unit test for this? you can just make a new file called test_hyundai that tests brand-specific things like this